src/verify.rs

```diff
@@ -46,7 +49,5 @@ pub(crate) async fn verify_local_checksum(
 }
 
 fn read_key_file(path: &str) -> Result<String> {
-    let verification_key =
-        fs::read_to_string(path).map_err(|e| anyhow!("Something went wrong: {:?}", e))?;
-    Ok(verification_key)
+    fs::read_to_string(path).map_err(|e| anyhow!("could not read file {}: {:?}", path, e))
```

Comment:
Maybe this could be done inside of the constructor of `SigstoreOpts`?

Reply:
This is about the verification key though (an argument that was present already), not directly related to the new fulcio & rekor new arguments.